src/layer/arm/convolutiondepthwise_3x3_int8.h

```diff
@@ -140,3 +145,438 @@ static void convdw3x3s2_int8_neon(const Mat &bottom_blob, Mat &top_blob, const M
+                    "vmlal.s8   q2, d8, d30         \n"// k4
+                    "vmlal.s8   q2, d9, d31         \n"// k5
+
+                    "pld        [%4, #128]          \n"
+                    "vld1.32    {d10-d11}, [%4]     \n"// r2
+                    "add        %4, #8              \n"
+                    "vext.8     d12, d10, d11, #1   \n"
+                    "vext.8     d13, d10, d11, #2   \n"
+                    
+                    "vdup.s8     d1, d26[6]         \n"
+                    "vdup.s8    d30, d26[7]         \n"
+                    "vdup.s8    d31, d27[0]         \n"             
+                    "vmlal.s8   q2, d10, d1         \n"// k6
+                    "vmlal.s8   q2, d12, d30        \n"// k7
+                    "vmlal.s8   q2, d13, d31        \n"// k8
+                    
+                    "vmovl.s16  q9, d4              \n"
+                    "vmovl.s16  q10, d5             \n"
+                    "vst1.32    {d18-d21}, [%1]!    \n"// sum0
+
+                    "subs       %0, #1              \n"
+                    "bne        0b                  \n"
+                    : "=r"(nn),             // %0
+                      "=r"(outptr0_s32),    // %1
+                      "=r"(r0),             // %2
+                      "=r"(r1),             // %3
+                      "=r"(r2)              // %4
+                    : "0"(nn),
+                      "1"(outptr0_s32),
+                      "2"(r0),
+                      "3"(r1),
+                      "4"(r2)
+                    : "cc", "memory", "q0", "q1", "q2", "q3", "q4", "q5", "q6", "q7", "q8", "q9", "q10", "q11", "q12", "q13", "q14", "q15"
+                );
+            }
+
+            for (; remain>0; remain--)
+            {
+                int sum = 0;
+
+                sum += (int)r0[0] * kernel[0];
+                sum += (int)r0[1] * kernel[1];
+                sum += (int)r0[2] * kernel[2];
+                sum += (int)r1[0] * kernel[3];
+                sum += (int)r1[1] * kernel[4];
+                sum += (int)r1[2] * kernel[5];
+                sum += (int)r2[0] * kernel[6];
+                sum += (int)r2[1] * kernel[7];
+                sum += (int)r2[2] * kernel[8];
+
+                *outptr0_s32 += sum;
+
+                r0++;
+                r1++;
+                r2++;
+                outptr0_s32++;
+            }   
+
+            r0 += 2;
+            r1 += 2;
+            r2 += 2;
+        }
+    }
+}
+
+static void convdw3x3s2_int8_neon(const Mat& bottom_blob, Mat& top_blob, const Mat& _kernel, const Option& opt)
+{
+    int w = bottom_blob.w;
+    int h = bottom_blob.h;
+    int inch = bottom_blob.c;
+
+    int outw = top_blob.w;
+    int outh = top_blob.h;
+    int outch = top_blob.c;
+
+    const int tailstep = w - 2*outw + w;
+
+    #pragma omp parallel for num_threads(opt.num_threads)
+    for (int p=0; p<outch; p++)
+    {
+        Mat out = top_blob.channel(p);
+
+        out.fill(0.f);
```

Comment:
not necessary for depthwise conv, the second instance

Reply:
It has been modified